src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -189,7 +189,7 @@ var TestRenderer = ReactFiberReconciler({
 
   useSyncScheduling: true,
 
-  getPublicInstance(inst) {
+  getPublicInstance(inst : Instance | TextInstance) {
```

Comment:
how do I say the return value is `Instance | TextInstance | createNodeMockReturnValue`?